### PR TITLE
Added README.md for public-docsite-v9

### PR DESCRIPTION
**Pull request by ValentinaKozlova**
Added README.md for public-docsite-v9

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/Avatar-mount_base.html">1319</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/Avatar-mount.html">1303</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/Button-mount_base.html">951</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/Button-mount.html">939</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/FluentProvider-mount_base.html">1593</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/FluentProvider-mount.html">1588</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/FluentProviderWithTheme-mount_base.html">634</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/FluentProviderWithTheme-mount.html">634</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">593</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">589</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">629</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">632</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>InfoButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/InfoButton-mount_base.html">566</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/InfoButton-mount.html">577</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/MakeStyles-mount_base.html">1870</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/MakeStyles-mount.html">1898</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/Persona-mount_base.html">3040</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/Persona-mount.html">2999</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/SpinButton-mount_base.html">6293</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26434/perf-test-react-components/SpinButton-mount.html">2530</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-alert</samp> <br /> <abbr title='bundle-size/Alert.fixture.js'>Alert</abbr>                                                                                       |  `85.912 kB`<br />`21.686 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `52.812 kB`<br />`14.689 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroup.fixture.js'>AvatarGroup</abbr>                                                                          |    `15.42 kB`<br />`6.167 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/AvatarGroupItem.fixture.js'>AvatarGroupItem</abbr>                                                                  |   `69.03 kB`<br />`18.984 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `33.354 kB`<br />`8.779 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |  `40.384 kB`<br />`10.127 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |  `38.043 kB`<br />`10.096 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |  `45.488 kB`<br />`11.501 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |  `49.184 kB`<br />`10.875 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `77.975 kB`<br />`22.496 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |   `72.92 kB`<br />`21.108 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |     `9.04 kB`<br />`3.807 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |   `10.805 kB`<br />`4.443 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `9.847 kB`<br />`4.159 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              |  `59.888 kB`<br />`16.652 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `187.539 kB`<br />`52.638 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `34.379 kB`<br />`11.322 kB` |
| <samp>react-persona</samp> <br /> <abbr title='bundle-size/Persona.fixture.js'>Persona</abbr>                                                                                 |  `58.849 kB`<br />`16.348 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |    `6.069 kB`<br />`2.053 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/DataGrid.fixture.js'>DataGrid</abbr>                                                                                 | `129.317 kB`<br />`35.207 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/PrimitivesOnly.fixture.js'>Table (Primitives only)</abbr>                                                            |  `42.971 kB`<br />`11.781 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableAsDataGrid.fixture.js'>Table as DataGrid</abbr>                                                                 | `127.563 kB`<br />`32.864 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSelectionOnly.fixture.js'>Table (Selection only)</abbr>                                                         |      `81 kB`<br />`20.124 kB` |
| <samp>react-table</samp> <br /> <abbr title='bundle-size/TableSortOnly.fixture.js'>Table (Sort only)</abbr>                                                                   |   `80.429 kB`<br />`19.97 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/06505b5c50c6deedcb36b29417cf35b77da85cfc'>06505b5c50c6deedcb36b29417cf35b77da85cfc</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: packages/react-components/react-button/stories/Button/ButtonShape.stories.tsx [hunk 1]
```diff
@@ -24,5 +24,29 @@ Shape.parameters = {
     description: {
       story: 'A button can be rounded, circular, or square.',
     },
+    source: {
+      code: `
+import * as React from 'react';
```
Comment (ling1726) [inline packages/react-components/react-button/stories/Button/ButtonShape.stories.tsx:29]: copying code like this isn't preferable IMO, if usage or features change and the doc samples are not updated then users who copy+paste are left with incorrect usage.

At least currently it's pretty obvious that users don't see the full code file in the show code area but can click on export to codesandbox to see the real code.

Comment (Hotell) [reply packages/react-components/react-button/stories/Button/ButtonShape.stories.tsx:29]: ^ that's very valid point

File: apps/public-docsite-v9/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# [Fluent UI V9 website](https://react.fluentui.dev/)
```
Comment (Hotell) [inline apps/public-docsite-v9/README.md:1]: # public-docsite-v9

This app is official documentation for Fluentui react-components published to https://react.fluentui.dev

File: apps/public-docsite-v9/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+**Fluent UI V9 React Components**
+
+Fluent UI is a collection of projects that represent the Fluent design language in code. This website helps document the components and styles that make up Fluent UI.
+
+## Build the website
+
+1. In `fluentui` folder run `yarn`.
```
Comment (Hotell) [inline apps/public-docsite-v9/README.md:9]: this is redundant step, if one has not installed deps you will get an error
```suggestion
```

File: packages/react-components/react-button/stories/Button/ButtonShape.stories.tsx [hunk 4]
```diff
@@ -24,5 +24,29 @@ Shape.parameters = {
...
+import { makeStyles, Button } from '@fluentui/react-components';
+
+const useStyles = makeStyles({
+  wrapper: {
+    columnGap: '15px',
+    display: 'flex',
+  },
+});
+
+export const Shape = () => {
```
Comment (Hotell) [inline packages/react-components/react-button/stories/Button/ButtonShape.stories.tsx:39]: this duplicity will probably lead into mistakes that might have negative effect on users. Unless this is done automatically behind the scenes by tooling IMHO this is a solution we should not go with.

**Review by ling1726 — APPROVED** [review]
Please update the PR title + description to avoid some misinformation :)

**Pull request merged**